TombEngine/Game/Lara/lara.cpp

```diff
@@ -461,7 +462,7 @@ void LaraAboveWater(ItemInfo* item, CollisionInfo* coll)
 
 	HandlePlayerBehaviorState(*item, *coll, PlayerBehaviorStateRoutineType::Control);
 	HandleLaraMovementParameters(item, coll);
-	AnimateItem(item);
+	AnimateItem(*item);
```

Comment:
As above, I think we must have an overload for `AnimateItem` which accepts pointers, not references.